client/src/pages/ManageBookings/useStyles.ts

```diff
@@ -0,0 +1,106 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  leftContainer: {
+    display: 'flex',
```

Comment:
Some of the styles within this file can be applied directly to the relevant MUI component

Reply:
@seblaifa3315 Please address the above comment

Reply:
@seblaifa3315 Please address comment before merging